sklearn/cluster/_mean_shift.py

```diff
@@ -38,20 +38,22 @@ def estimate_bandwidth(X, quantile=0.3, n_samples=None, random_state=0,
     X : array-like of shape (n_samples, n_features)
         Input points.
 
-    quantile : float, default 0.3
+    quantile : float, default=0.3
         should be between [0, 1]
         0.5 means that the median of all pairwise distances is used.
+        The default is 0.3
```

Comment:
no need

Reply:
I've edited the descriptions to remove the defaults I had added.